Kernel/API/Syscall.h

```diff
@@ -132,7 +132,9 @@ namespace Kernel {
     S(dump_backtrace)         \
     S(dbgputch)               \
     S(dbgputstr)              \
-    S(watch_file)             \
+    S(create_iwatcher)        \
+    S(iwatcher_add_watch)     \
+    S(iwatcher_remove_watch)  \
```

Comment:
Did you run out of identifier juice while writing these?
Why not `inode_watcher`? (+Everywhere else)

Reply:
Hmmm...
`inode_watcher_add` is too close to `create_inode_watcher`...idk, maybe `iwatcher` is fine then